src/WorkflowManager/MonaiBackgroundService/Program.cs

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Will this be deployed in a separate container? Any reason not to have the service as part of WM or TM?

Reply:
No you are right this should have been called from WM.